rolling/distribution.yaml

```diff
@@ -917,6 +917,25 @@ repositories:
       url: https://github.com/carologistics/clips_vendor.git
       version: main
     status: maintained
+  cob_common:
+    doc:
+      type: git
+      url: https://github.com/4am-robotics/cob_common.git
+      version: foxy
+    release:
+      packages:
+      - cob_actions
+      - cob_msgs
+      - cob_srvs
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/cob_common-release.git
```

Comment:
Just as an FYI, the humble release of this is in a different release repository.  While that is allowed, you might find it confusing, so the recommendation is to use the same release repository for all releases.